sdk/template/azure-template-second/inc/azure/template/second/dll_import_export.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief This file defines a macro for DLL export.
+ */
+
+// For explanation, see the comment in azure/core/dll_import_export.hpp
+
+#pragma once
+
+#if defined(AZ_TEMPLATE_DLL) || (0 /*@AZ_TEMPLATE_DLL_INSTALLED_AS_PACKAGE@*/)
```

Comment:
There MACROS need to be named specific to the package name you choose.  i.e.
```suggestion
#if defined(AZ_TEMPLATE_SECOND_DLL) || (0 /*@AZ_TEMPLATE_SECOND_DLL_INSTALLED_AS_PACKAGE@*/)
```